README.md

```diff
@@ -268,6 +305,43 @@ and replace it with '/public/'.
 
 _Default_: `''`
 
+#### runtimeCaching [Array&#x27e8;Object&#x27e9;]
+Used to configure runtime caching for dynamic content. If you use this option, the `sw-toolbox`
```

Comment:
Change from "Used to configure runtime caching..." to "Configures runtime caching..."